sql/2019/05_Third_Parties/05_01.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Percentage of pages that include at least one third party resource.
+SELECT
+  COUNT(*) as numberOfPages,
```

Comment:
There are some conventions in the [Analyst wiki](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide) that can be applied here. (eg, uppercase `AS`)

I'd also like to see `COUNT(0)` used rather than `COUNT(*)` as a micro optimization since we're counting the rows and the fields don't matter.

Reply:
great points, done!